### ISSUE TITLE
[OTHER]: Missing margin 

### ISSUE THREAD

**Issue by root-0101** [Nishitbaria/Dev-Exchange#78]
### What would you like to share or ask?

There is missing top margin in the profile posts card 

![Screenshot 2024-05-31 231956](https://github.com/Nishitbaria/Dev-Exchange/assets/142779809/7b0dabe9-990b-4f12-9d3c-70046f286e5d)

### Checklist

- [X] I have checked the existing issues
- [X] I have read the [Contributing Guidelines](https://github.com/Nishitbaria/Dev-Exchange/blob/main/CONTRIBUTING.md)
- [X] I am willing to work on this issue (optional)
